docs/zh-cn/overview/faq.md

```diff
@@ -36,20 +36,21 @@ undolog序列化配置为jackson时，jackson版本需要为2.9.9+
 
 **A:** 
   1. 脏数据需手动处理，根据日志提示修正数据或者将对应undo删除（可自定义实现FailureHandler做邮件通知或其他）
-  2. 关闭回滚时undo镜像校验，不推荐该方案。
+  2. 关闭回滚时undo镜像校验，不推荐该方案。  
     注：建议事前做好隔离保证无脏数据
 
 ********
 ### Q: 为什么分支事务注册时, 全局事务状态不是begin ？
 
 **A:**  
-    异常：Could not register branch into global session xid = status = Rollbacked（还有Rollbacking、AsyncCommitting等等二阶段状态） while expecting Begin
-    描述：分支事务注册时，全局事务状态需是一阶段状态begin，非begin不允许注册。属于seata框架层面正常的处理，用户可以从自身业务层面解决。
-    有以下几种情况会出现该异常（可继续补充）
+* 异常：Could not register branch into global session xid = status = Rollbacked（还有Rollbacking、AsyncCommitting等等二阶段状态） while expecting Begin
+* 描述：分支事务注册时，全局事务状态需是一阶段状态begin，非begin不允许注册。属于seata框架层面正常的处理，用户可以从自身业务层面解决。
+* 出现场景（可继续补充）
+```
```

Comment:
![image](https://user-images.githubusercontent.com/827112/70154456-7ab48d00-16eb-11ea-9ad3-601e3d6b55f5.png)
Why add ```  ``` ?  The scroll bar will appear

Reply:
To wrap